openmetadata-service/src/test/java/org/openmetadata/service/resources/services/DatabaseServiceResourceTest.java

```diff
@@ -15,18 +15,14 @@
 
 import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
 import static javax.ws.rs.core.Response.Status.OK;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
@EsmaeilChitgar please avoid wild-card imports

Reply:
Will avoid wild-card imports. Thanks.

Reply:
Thanks for your feedback. I'll apply it as soon as possible.